<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camera</class>
 <widget class="QMainWindow" name="camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,7">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>列：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>行：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>大小：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>图片浏览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>检测结果</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>标定结果</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,10">
      <item>
       <widget class="QListWidget" name="ListWidget_imgs"/>
      </item>
      <item>
       <widget class="QLabel" name="imgshow">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="newpro_flag">
    <property name="title">
     <string>新建项目</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>打开项目</string>
    </property>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>保存项目</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>删除图片</string>
    </property>
    <addaction name="actiondelet"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>设置棋盘</string>
    </property>
    <addaction name="actionset"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>检测</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>标定</string>
    </property>
    <addaction name="action1"/>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="title">
     <string>导出结果</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_10">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="inputPushButton">
    <property name="title">
     <string>添加图片</string>
    </property>
    <addaction name="inputbuttonimg"/>
   </widget>
   <addaction name="newpro_flag"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="inputPushButton"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
   <addaction name="menu_9"/>
   <addaction name="menu_10"/>
  </widget>
  <action name="actionix">
   <property name="text">
    <string>ix</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>新建项目</string>
   </property>
  </action>
  <action name="actioninput">
   <property name="text">
    <string>添加图片</string>
   </property>
  </action>
  <action name="inputbuttonimg">
   <property name="text">
    <string>选择图片</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>开始标定</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>保存结果</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>开始检测</string>
   </property>
  </action>
  <action name="actiondelet">
   <property name="text">
    <string>删除图片</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>保存项目</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>打开项目</string>
   </property>
  </action>
  <action name="actionset">
   <property name="text">
    <string>设置棋盘</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
